packages/react/src/tests/e2e.test.tsx

```diff
@@ -13,8 +14,9 @@ import {
 
 const RemoteHelloWorld = createRemoteReactComponent<
   'HelloWorld',
-  {name: string}
->('HelloWorld');
+  {name: string | RemoteFragment}
+>('HelloWorld', {fragmentProps: ['name']});
+RemoteHelloWorld.displayName = 'HelloWorld';
```

Comment:
Sorry, I meant that we should do this in `createRemoteReactComponent`, when we create an actual function component.

Reply:
A minor issue with this is that `createRemoteReactComponent` doesn't always return object. If there is no fragmentProps, it returns a string. A string cannot set `displayName`.

Reply:
Yup, we should only do this in the case where we are creating a function component for you (and not doing it probably means we have some unhelpful component names in the tree)